.github/workflows/add-bugs-to-project.yml

```
@@ -0,0 +1,19 @@
+name: Assign Bugs to Project
+
+on:
+    issues:
+        types: [labeled]
+env:
+    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+    assign_to_project:
+        runs-on: ubuntu-latest
+        name: Assign bug issue to Bug Management project
+        steps:
+            - name: Assign issues with type:bug label to Firefly - Bug Management project
+              uses: srggrs/assign-one-project-github-action@1.2.0
```

```suggestion
              uses: srggrs/assign-one-project-github-action@4fc2b23bdaaac08e64dcc590a6e138d2f9b8c86e
```
For third-party actions that deal with GitHub tokens, the best practice is to lock it to a specific commit SHA. Release tags, on the other hand, can be changed by the action's owner without our knowledge